docs/script.js

```diff
@@ -1,27 +1,250 @@
+  // Set image preview
+  imgPreview.setAttribute('src', projects[ProjectNum]['featured image']);
+  // Set text content
+  textPreview.textContent = projects[ProjectNum].description;
+  // Set list of tools used
+  const array = projects[ProjectNum].technologies;
+  for (let index = 0; index < array.length; index += 1) {
+    const element = document.createElement('li');
+    element.textContent = array[index];
+    listOfTechnologies.appendChild(element);
+  }
+  // Set buttons links
+  link.setAttribute('href', projects[ProjectNum]['link to live version']);
+  link.setAttribute('target', '_blank');
+  link2.setAttribute('href', projects[ProjectNum]['link to source']);
+  link2.setAttribute('target', '_blank');
 }
+// Quit the details popup
+function quitDetails() {
+  document.getElementById('stylesheet').disabled = false;
+  btnDetails[0].removeAttribute('style');
+  document.querySelectorAll('.positionQBtn, .but, .deco li').forEach((item) => {
+    item.remove();
+  });
+  document.querySelectorAll('*:not(.menu-var *,#menu-v)').forEach((e) => e.classList.remove('hidden'));
+  document.getElementById('section-2').removeAttribute('style');
+  document.getElementById('section-4').classList.remove('blur-bg');
+  document.querySelector('#section-4 > article').classList.remove('articleStyle');
+  header3.classList.remove('headerMobile');
+  imgPreview.removeAttribute('style');
+  textPreview.classList.remove('p-style');
+  listOfTechnologies.classList.remove('deco');
+  listOfTechnologies.removeAttribute('style');
+  document.querySelector('#section-4 > article button').removeAttribute('style');
 
-function quitMenu() {
-  quitbtn.classList.add('hidden');
-  menuVar.forEach((e) => e.classList.add('hidden'));
-  content.forEach((e) => e.classList.remove('hidden'));
-  navVar.removeAttribute('style');
-  aTags.forEach((e) => e.removeAttribute('style'));
+  generatePost(4, document.querySelector('#section-4 article'), document.querySelector('#section-4 article'));
+}
+// Creates a quit button on the element passed, with the style and function.
+function createQuitBtn(element) {
+  const qBtn = document.createElement('img');
+  qBtn.setAttribute('src', 'sprites/quitbtn.svg');
+  qBtn.classList.add('positionQBtn');
+  element.appendChild(qBtn);
+  qBtn.addEventListener('click', quitDetails);
+  return qBtn;
+}
+// Creates a button
+function createButton(element) {
+  const button = document.createElement('button');
+  button.setAttribute('type', 'button');
+  button.classList.add('buton-effects', 'text');
+  element.appendChild(button);
+  return button;
+}
+// Create button icons
+function createIcon(container) {
+  const img = document.createElement('img');
+  container.appendChild(img);
+  return img;
+}
+
+function createAlink(container) {
+  const a = document.createElement('a');
+  container.appendChild(a);
+  return a;
+}
+// Determine which card was clicked
+function cardSelected(event) {
+  return event.path[0].id;
+}
```

Comment:
Great job! Modal popup data is populated with the data from the clicked project 🔥